SimpleSAMLphpAuth.php

```diff
@@ -0,0 +1,97 @@
+<?php
+# Licensed under the MIT license
+
+/**
+ * Sample Auth plugin
+ */
+class SimpleSAMLphpAuthPlugin extends MantisPlugin  {
+	/**
+	 * A method that populates the plugin information and minimum requirements.
+	 * @return void
+	 */
+	function register() {
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
+		$this->page = 'config_page';
+
+		$this->version = '0.1';
+		$this->requires = array(
+			'MantisCore' => '2.4.0',
+		);
+
+		$this->author = 'Erwann PENET';
+		$this->contact = '';
+		$this->url = 'https://github.com/mantisbt-plugins/SimpleSAMLphpAuth';
+	}
+
+	/**
+	 * plugin hooks
+	 * @return array
+	 */
+	function hooks() {
+		$t_hooks = array(
+			'EVENT_AUTH_USER_FLAGS' => 'auth_user_flags',
+		);
+
+		return $t_hooks;
+	}
+
+	function config() {
+		return array(

```

Comment:
How about adding a `test_users` that can be set to a comma separated usernames to only enable the plugin for such users for testing.  If not set/empty, then plugin applies to all.  That enables the admin to test on a specific accounts before playing around with configs which may lock everybody out.